entrypoint.js

```diff
@@ -1,84 +1,28 @@
-const axios = require("axios");
-const fs = require("fs");
-const _ = require("lodash");
-const querystring = require("querystring");
-const { argv } = require("yargs");
+const axios = require('axios');
+const message = require('./message.js');
 
 const REQUIRED_ENV_VARS = [
   "GITHUB_EVENT_PATH",
-  "GITHUB_REPOSITORY",
-  "GITHUB_WORKFLOW",
-  "GITHUB_ACTOR",
-  "GITHUB_EVENT_NAME",
-  "GITHUB_ACTION",
-  "SLACK_WEBHOOK"
+  "SLACK_WEBHOOK",
```

Comment:
Admitting that the Action can be run locally, I would recommend to keep those variables as required. GitHub always sets them at run time (https://help.github.com/en/actions/automating-your-workflow-with-github-actions/using-environment-variables#default-environment-variables). Or, is there a reason you removed them ?

Reply:
Just wanted to leave the required to the minimal. and act more on fallbacks.
Check required only if it will like, throw error on the app or something..
ill add them back anyway.

Reply:
Now that you're saying this, I just realized I was completely wrong. You're right. Moreover, we would not really need those variables anyway, since they were only used in the default notification string. Do you agree if we remove them again ? :-)

Reply:
Delivered.
_unlike my 🍕_